.github/workflows/continuous-integration.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         run: php7.2 $(which composer) validate --strict
 
       - name: "Cache dependencies installed with composer"
-        uses: actions/cache@v1.0.0
```

Comment:
I was hoping to be able to use https://dependabot.com/github-actions/ soon!

🤓

Reply:
oh awesome!

Reply:
looks like it already exists.
https://github.com/dependabot/dependabot-core/blob/master/github_actions/lib/dependabot/github_actions.rb
try using `github_actions` as the `package_manager`?